src/coreComponents/mesh/generators/VTKMeshGenerator.hpp

```diff
@@ -23,6 +23,7 @@
 #include "mesh/generators/VTKUtilities.hpp"
 
 #include <vtkDataSet.h>
+#include <string>
```

Comment:
We don't use `std::string`, we use `geosx::string` (as a matter of fact a `typedef` to `std::string`).
It's defined in `src/coreComponents/common/DataTypes.hpp`. (even if we still use some `std::string` or `std::to_string` here and there.)